Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -96,12 +127,10 @@ class BillingFormTextFieldView: UIView {
 
     private func updateTextFieldContainer(style: CellTextFieldStyle) {
         let borderColor = !(style.error?.isHidden ?? true) ?
-        style.textfield.errorBorderColor.cgColor :
-        style.textfield.normalBorderColor.cgColor
-
-        textFieldContainer.layer.borderColor = borderColor
-        textFieldContainer.layer.cornerRadius = style.textfield.cornerRadius
-        textFieldContainer.layer.borderWidth = style.textfield.borderWidth
+        style.textfield.borderStyle.errorColor.cgColor :
+        style.textfield.borderStyle.normalColor.cgColor
+        textFieldContainer.update(with: style.textfield.borderStyle)
+        textFieldContainer.borderLayer.strokeColor = borderColor
```

Comment:
I think this method should be moved inside the BorderView so we don't expose the `borderLayer` directly?

```suggestion
        textFieldContainer.updateBorderColor(to: borderColor)
```